spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/main/java/com/example/Trader.java

```diff
@@ -50,6 +51,9 @@
 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MMM-dd HH:mm:ss z")
 	private List<java.sql.Timestamp> modifiedOn;
 
+	@Interleaved(lazy = true)
+	private List<Trade> trades;
```

Comment:
Should we leave a comment that it should be left uninitialized? Or should we just do that fix you suggested where we initialize it if the list is empty?

Reply:
I haven't really looked into that possible fix. It's not common to initialize like that in entities. So, the comment might be distracting.

Reply:
Hm yeah makes sense. Maybe making this mistake is fairly rare.